### PR TITLE
What the format is of the newsletter

### PR DESCRIPTION
**Pull request by bvhme**

### CODE REVIEW

File: format.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+We write a weekly newsletter on awesome things. This is mainly these things:
+- About how we are improving our environment creating a more liveable world for humans.
+- Positive turn of events, that doesn’t stress negativity too much. Sometimes you can use a non-positive article but just highlight the cool things about it.
+- Current, maximally one week old on the sending date (maybe more if it's very cool).
+- It has to have a cool picture that we can use in the Newsletter.
+- A video is a definite plus.
+
+Every newsletter contains the following:
+
+- A subject line that is exciting
```
Comment (thomasdebeus) [inline format.md]: that makes the receiver to open the newsletter.

File: format.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- A short emoji teaser to show people who have the email content preview switched on
+- 3-5 awesome things that were happening this week
+- An awesome gif (does not nessecerilly have to be related with the content)
+
+This is how the newsletter's contents is build up:
+
+1. The emoji teaser.
+2. The view this newsletter in the browser link.
+3. The header image, logo and title.
+4. Salutation
```
Comment (thomasdebeus) [inline format.md]: possible with a name merged tag to give a more personal feel.

File: format.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+5. In short bullet points what the takaways are of this weeks newsletter. (one bullet point per article)
+6. Every article with:
+  - An image from and linking to the original source of the story.
+  - A good headline.
```
Comment (thomasdebeus) [inline format.md]: hyperlinked to original article.

**Comment by bvhme** [comment]
@thomasdebeus feel free to edit the file as well.

Instead of FNAME I would say, 'Name merge tags', 'subscriber's name' or something less Mailchimp specific

File: format.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+---
+
+# The format of a How We Have Been Saving The Planet This Week newsletter
+
+We write a weekly newsletter on awesome things. This is mainly these things:
+- About how we are improving our environment creating a more liveable world for humans.
```
Comment (thomasdebeus) [inline format.md:8]: We write about, and share articles that are about how we are improving our environment creating a more liveable world for humans.

File: format.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Every newsletter contains the following:
+
+- A subject line that is exciting and makes the receiver to open the newsletter.
+- A short emoji teaser to show people who have the email content preview switched on
+- 3-5 awesome things that were happening this week
+- An awesome gif (does not nessecerilly have to be related with the content)
+
+This is how the newsletter's contents is build up:
+
+1. The emoji teaser.
```
Comment (fdeerenberg) [inline format.md]: hoe ziet dat er uit?

**Comment by bvhme** [comment]
@thomasdebeus is this good to merge?

**Comment by thomasdebeus** [comment]
Yeah cool

**Pull request merged**